usr/lib/common/key.c

```diff
@@ -6346,6 +6420,54 @@ CK_RV aes_validate_attribute(STDLL_TokData_t *tokdata, TEMPLATE *tmpl,
     }
 }
 
+//
+//
+CK_RV aes_xts_validate_attribute(STDLL_TokData_t *tokdata, TEMPLATE *tmpl,
```

Comment:
Same here, this also seems to be a duplication of aes_validate_attribute with the exception of a few places.
Also modify aes_validate_attribute and pass a keytype parameter to it.